CHANGES.md

```diff
@@ -1,3 +1,7 @@
+### 1.1 ()
```

Comment:
I guess this should be part of the 1.0 release, that I'm planning to issue as soon as the work on LSP is somewhat ready, and the installation of the emacs mode is done properly.

Reply:
Oh, I saw the date thus I assumed the release was done. Amending.

Reply:
Yes sorry, I was planning to release today, and I changed my mind with this single-opam-package thing!